index.html

```diff
@@ -161,6 +161,7 @@ <h4>All the tutorials can be run locally or with <a href="https://www.confluent.
             <div>Connect data sources & sinks</div>
           </div>
           <ul>
+            <li><a href="connect-sink-timestamp/ksql.html">Time conversion and Kafka Connect JDBC sink</a></li>
```

Comment:
1. Move below `Add a key to a stream`?
2. To keep the titles short, WDYT:

```suggestion
            <li><a href="connect-sink-timestamp/ksql.html">Convert timezone before sending to a database</a></li>
```

Reply:
My leaning would be to put it on the stop simply because it's new, but I don't feel strongly about that :) @ybyzek why add it below "add a key to a stream"?

Reply:
My leaning is that we should promote new KTs on Confluent Developer, and on the KT front page we order the KTs logically.  So adding it below `add a key to a stream` seemed logical to show source before sink.  WDYT?